include/libc/stdarg.h

```diff
@@ -1,8 +1,6 @@
 #ifndef STDARG_H
 #define STDARG_H
```

Comment:
```suggestion
#ifndef LIBC_STDARG_H
#define LIBC_STDARG_H
```
Can you do this for all the .h files in libc to match strings.h and copy the format of ultra64